src/Microsoft.VisualStudio.AppDesigner/interop/NativeMethods.vb

```diff
@@ -16,10 +16,10 @@ Namespace Microsoft.VisualStudio.Editors.AppDesInterop
         Private Const VB_COMPILER_GUID As String = "019971d6-4685-11d2-b48a-0000f87572eb"
         Public Shared ReadOnly VBCompilerGuid As Guid = New Guid(VB_COMPILER_GUID)
 
-        '/ <summary>
-        '/     Handle type for HDC's that count against the Win98 limit of five DC's.  HDC's
-        '/     which are not scarce, such as HDC's for bitmaps, are counted as GDIHANDLE's.
-        '/ </summary>
+        ' <summary>
+        '     Handle type for HDC's that count against the Win98 limit of five DC's.  HDC's
+        '     which are not scarce, such as HDC's for bitmaps, are counted as GDIHANDLE's.
+        ' </summary>
```

Comment:
Should be triple slash right?

Reply:
I believe so, yes. Will update.

Reply:
This is spread throughout the changes not this one location.

Reply:
The others are `<devdoc>` which are not VB standard as far as I know.